packages/select/package.json

```diff
@@ -0,0 +1,71 @@
+{
+  "name": "@blueprintjs/select",
+  "version": "1.0.0",
+  "description": "Components related to selecting items from a list",
+  "main": "dist/index.js",
+  "module": "dist/esm/index.js",
+  "typings": "dist/esm/index.d.ts",
+  "style": "dist/blueprint-select.css",
+  "unpkg": "dist/select.bundle.js",
+  "scripts": {
+    "compile": "npm-run-all -p 'compile:*'",
+    "compile:esm": "tsc -p ./src",
+    "compile:cjs": "tsc -p ./src/tsconfig.cjs.json",
+    "compile:css": "node-sass --importer ../../node_modules/node-sass-package-importer/dist/cli.js --recursive src --output dist/ --source-map true",
```

Comment:
use the new `sass-compile` script which gilad recently added (pull in master, see the other packages)